src/display/api.js

```diff
@@ -99,12 +99,11 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
   createPDFNetworkStream = pdfNetworkStreamFactory;
 }
 
+/* eslint-disable max-len */
 /**
- * @typedef {
- *   Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array |
- *   Int32Array | Uint32Array | Float32Array | Float64Array
- * } TypedArray
+ * @typedef {Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array | Uint32Array | Float32Array | Float64Array} TypedArray
  */
+/* eslint-enable max-len */
```

Comment:
The reason that we enforce a maximum line-length, even for comments, is related to readability. Hence disabling of the ESLint `max-len` rule should basically never happen (even more so now that Prettier is being used).

(Furthermore, note that we always prefer using `// eslint-disable-line` or `// eslint-disable-next-line`, since this sort of blanket disabling increases the risk that one forgets to re-enable a rule.)

---

However, based on a very quick test there seems to be a *much* better solution to this problem:
```javascript
/**
 * @typedef { Int8Array | Uint8Array | Uint8ClampedArray |
 *            Int16Array | Uint16Array |
 *            Int32Array | Uint32Array | Float32Array |
 *            Float64Array
 * } TypedArray
 */
```

Reply:
I have edited as suggested. Thanks.